meta-mender-sunxi/recipes-bsp/u-boot/files/0002-env-Kconfig-remove-defaults-for-SUNXI.patch

```diff
@@ -1,43 +1,36 @@
-From 62081aeabe64ee25dfe90c5fcff8e1268bbd382a Mon Sep 17 00:00:00 2001
+From 6679c047066ff17e666153e9614efe2aac58f4e3 Mon Sep 17 00:00:00 2001
 From: Mirza Krak <mirza.krak@northern.tech>
 Date: Mon, 6 Aug 2018 21:21:09 +0200
-Subject: [PATCH 1/2] env: Kconfig: remove defaults for SUNXI
+Subject: [PATCH] env: Kconfig: remove defaults for SUNXI
 
 That is ENV_IS_IN_FAT and the default offset. Mender build will set
 up these correctly
 
 Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
 ---
- env/Kconfig | 9 ---------
- 1 file changed, 9 deletions(-)
+ env/Kconfig | 2 --
+ 1 file changed, 2 deletions(-)
 
 diff --git a/env/Kconfig b/env/Kconfig
-index a3c6298273..b14a5b1789 100644
+index f23be00..f89a01f 100644
 --- a/env/Kconfig
 +++ b/env/Kconfig
-@@ -55,7 +55,6 @@ config ENV_IS_IN_FAT
+@@ -56,7 +56,6 @@ config ENV_IS_IN_FAT
  	bool "Environment is in a FAT filesystem"
  	depends on !CHAIN_OF_TRUST
  	default y if ARCH_BCM283X
 -	default y if ARCH_SUNXI && MMC
  	default y if MMC_OMAP_HS && TI_COMMON_CMD_OPTIONS
  	select FS_FAT
  	select FAT_WRITE
-@@ -439,14 +438,6 @@ config ENV_EXT4_FILE
-
- if ARCH_SUNXI
-
--config ENV_OFFSET
--	hex "Environment Offset"
--	depends on !ENV_IS_IN_UBI
--	depends on !ENV_IS_NOWHERE
--	default 0x88000 if ARCH_SUNXI
--	help
--	  Offset from the start of the device (or partition)
--
- config ENV_SIZE
- 	hex "Environment Size"
+@@ -438,7 +437,6 @@ config ENV_OFFSET
```

Comment:
Can not leave this one unfortunately because it will default to ENV_OFFSET="", and as you might have noticed that meta-mender complains about this.

Reply:
Oh right, then it complain that ENV_OFFSET is empty and could be 0x800000 so should I drop it?